influx/templates_test.go

```diff
@@ -62,10 +62,10 @@ func TestTemplateReplace(t *testing.T) {
 		},
 		{
 			name:  "select with parameters and aggregates",
-			query: `SELECT mean($field) FROM "cpu" WHERE $tag = $value GROUP BY $tag`,
+			query: `SELECT mean(:field:) FROM "cpu" WHERE :tag: = :value: GROUP BY :tag:`,
```

Comment:
Why'd these change from `$` to `:` in this PR?

Reply:
Cleaning up issues from previous code.  Originally, it was $ and we changed it to slack-y syntax.  I wanted to make sure the tests didn't look odd as compared to what we really use.